pkg/apis/serving/v1alpha2/openapi_generated.go

```diff
@@ -50,11 +51,6 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 		"./pkg/apis/serving/v1alpha2.TransformerSpec":         schema_pkg_apis_serving_v1alpha2_TransformerSpec(ref),
 		"./pkg/apis/serving/v1alpha2.TritonSpec":              schema_pkg_apis_serving_v1alpha2_TritonSpec(ref),
 		"./pkg/apis/serving/v1alpha2.XGBoostSpec":             schema_pkg_apis_serving_v1alpha2_XGBoostSpec(ref),
-		//TODO: `make generate` will remove this due to knative version upgrade but is used by python client
```

Comment:
v1alpha2 is no longer generated, but let's keep these as is and do not remove it.